src/Server.php

```diff
@@ -90,6 +90,25 @@ public function startBrokerSession()
 
         $this->brokerId = $this->validateBrokerSessionId($sid);
     }
+		
+	/**
+	 * Get session ID from header Authorization or from $_GET/$_POST
+	 */
+	protected function getBrokerSessionID(){
+		$headers = getallheaders();
+		
+		if (isset($headers['Authorization'])){
+			return $headers['Authorization'];
+		}
+		if (isset($_GET['sso_session'])) {
+			return $_GET['sso_session'];
+		}
+		if (isset($_POST['sso_session'])) {
+			return $_POST['sso_session'];
+		}
+		
+		return FALSE;
```

Comment:
Please adhere to [PSR2](http://www.php-fig.org/psr/psr-2/) and use `false`, not `FALSE`.